enclave/core/calls.c

```diff
@@ -260,35 +260,76 @@ static oe_result_t _handle_call_enclave_function(uint64_t arg_in)
 {
     oe_call_enclave_function_args_t args, *args_ptr;
     oe_result_t result = OE_OK;
-    oe_enclave_func_t f = NULL;
+    oe_ecall_func_t func = NULL;
```

Comment:
@gupta-ak Can you take a close look at this function to make sure we are handling all corner cases?